tests/regression/57-floats/06-library_functions.c

```diff
@@ -2,12 +2,13 @@
 #include <assert.h>
 #include <math.h>
 #include <float.h>
+#include <complex.h>

```

Comment:
Is this header actually required?

Reply:
ah, no, just missed that when I was testing something